### PR TITLE
IBM Cloud node stop and start added

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

File: ocs_ci/utility/ibmcloud.py [hunk 1]
```diff
@@ -337,6 +338,52 @@ class IBMCloud(object):
     Wrapper for Ibm Cloud
     """
 
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:347]: Add `Raises` section mention: `ValueError` there as this is one you raises in some cases ;)

File: ocs_ci/utility/ibmcloud.py [hunk 2]
```diff
@@ -337,6 +338,52 @@ class IBMCloud(object):
...
+
+        """
+        if not nodes:
+            raise ValueError("No nodes found to start")
+
+        node_names = [n.name for n in nodes]
+        self.ibmcloud.restart_nodes(nodes)
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:354]:
```suggestion
        self.restart_nodes(nodes)
```

Did you forgot to remove `ibmcloud` here?

File: ocs_ci/utility/ibmcloud.py [hunk 3]
```diff
@@ -337,6 +338,56 @@ class IBMCloud(object):
...
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
+        
+        Raises:
+            ValueError if the list of nodes is empty
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:350]:
```suggestion
            ValueError: if the list of nodes is empty

```
one empty line after this section, updated in suggestion and colon after the exception

File: ocs_ci/utility/ibmcloud.py [hunk 4]
```diff
@@ -337,6 +338,56 @@ class IBMCloud(object):
...
+    def stop_nodes(self, nodes, wait=True):
+        """
+        Stop nodes on IBM Cloud
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to stop, False otherwise
+
+        Raises:
+            ValueError if the list of nodes is empty
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:374]:
```suggestion
            ValueError: if the list of nodes is empty

```

File: tests/functional/object/mcg/ui/test_namespace_store.py [hunk 5]
```diff
@@ -4,7 +4,11 @@
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs import constants
 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import black_squad, runs_on_provider
+from ocs_ci.framework.pytest_customization.marks import (
+    black_squad,
```
Comment (petr-balogh) [inline tests/functional/object/mcg/ui/test_namespace_store.py:8]:
```suggestion
    black_squad,
```

File: tests/functional/object/mcg/ui/test_namespace_store.py [hunk 6]
```diff
@@ -4,7 +4,11 @@
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs import constants
 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import black_squad, runs_on_provider
+from ocs_ci.framework.pytest_customization.marks import (
+    black_squad, 
+    runs_on_provider,
+    skipif_ibm_cloud_managed,
+    )
```
Comment (petr-balogh) [inline tests/functional/object/mcg/ui/test_namespace_store.py:11]:
```suggestion
)
```

File: ocs_ci/utility/ibmcloud.py [hunk 7]
```diff
@@ -337,6 +338,56 @@ class IBMCloud(object):
...
     """
 
+    def start_nodes(self, nodes, wait=True):
+        """
+        Start nodes on IBM Cloud.
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to start, False otherwise
+
```
Comment (petr-balogh) [inline ocs_ci/utility/ibmcloud.py:348]: you have some trailing whitespaces here in between section

**Review by petr-balogh — DISMISSED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Review by fbalak — APPROVED** [review]

**Review by paraggit — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

**Review by keemano — APPROVED** [review]

**Pull request merged**